frameworks/Go/go-raw/src/hello/hello.go

```diff
@@ -48,11 +48,17 @@ const (
 	connectionString   = "benchmarkdbuser:benchmarkdbpass@tcp(localhost:3306)/hello_world?interpolateParams=true"
```

Comment:
It would be good to update the comment to mention that there are two benchmarks now.